packages/block-library/src/navigation/style.scss

```diff
@@ -106,6 +106,45 @@ $navigation-icon-size: 24px;
 			height: inherit;
 		}
 	}
+
+	// Custom properties for layout style cascade: default values.
+	--layout-justification-setting: flex-start;
```

Comment:
Can we make these CSS variables more specific to the navigation block or something? They feel a bit too generic.

Reply:
Done! I prefixed them with "navigation".